ops/test/swap/swap.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
```

Comment:
I know swap is the Azure mechanism but could we come up with a more generic directory name? (Promote?)